scout/build/case.py

```diff
@@ -1,7 +1,7 @@
 import logging
 from datetime import datetime
 
-from scout.constants import CUSTOM_CASE_REPORTS
+from scout.constants import CUSTOM_CASE_REPORTS, GENOME_BUILD_37, GENOME_BUILD_38
```

Comment:
This is also part or another branch perhaps?

Reply:
Fixed